charts/datafold/charts/storage-worker/values.yaml

```diff
@@ -17,6 +17,7 @@ worker:
   tasks: 200
```

Comment:
```suggestion
  tasks: 1
```

If I understand the meaning of this setting?

Reply:
@anatoly-scherbakov : Works. I'll update.